content/docs/command-reference/plots/index.md

```diff
@@ -37,18 +37,26 @@ This type of metrics files are created by users, or generated by user data
 processing code, and can be defined in `dvc.yaml` (`plots` field) for tracking
 (optional).
 
-DVC generates plots as HTML files that can be open with a web browser. These
-HTML files use [Vega-Lite](https://vega.github.io/vega-lite/). Vega-Lite is a
-declarative grammar for defining plots using JSON. The plots can also be saved
-as SVG or PNG image filed from the browser.
+DVC can work with two types of plots files:
 
-In contrast to `dvc metrics`, these metrics should be stored as data series.
-Unlike its `dvc metrics` counterpart, `dvc plots diff` cannot calculate numeric
-differences between the metrics in different experiments.
+1. Data series files, which can be JSON, YAML, CSV or TSV.
+2. Image files, like JPEG, GIF, or PNG.
+
+DVC generates plots as static HTML webpages that can be open with a web browser.
+They can also be saved as SVG or PNG image files from the browser.
+
+Data-series plots utilize [Vega-Lite](https://vega.github.io/vega-lite/) for
+rendering (declarative JSON grammar for defining graphics). Image-type plots are
+rendered using `<img>` tags directly.
 
 ### Supported file formats
 
-Plot metrics can be organized as data series in JSON, YAML 1.2, CSV, or TSV
+Image type plots are included in HTML as is, without additional processing.
+
+> ⚠️ When using image plots we recommend to store them in DVC cache and not in
+> Git history, to prevent the repository size from bloating.
+
+Data series plots can be organized as data series in JSON, YAML 1.2, CSV, or TSV
```

Comment:
```suggestion
Image-type plots are included in HTML as-is, without additional processing.

> We recommend to track these source image files with DVC instead of Git,
> to prevent the repository from bloating.

Structured plots can be read from JSON, YAML 1.2, CSV, or TSV
```